src/files/files.go

```diff
@@ -0,0 +1,341 @@
+package files
+
+import (
+	"bufio"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+const (
+	colorYellow = "\033[33m"
+	colorReset  = "\033[0m"
+	messageTag  = "message:"
+)
+
+var (
+	changeLogDefaultFile = "CHANGELOG.md"
+)
+
+type Logger interface {
+	Info(s string, args ...interface{})
+	Error(s string, args ...interface{})
+	Warn(s string, args ...interface{})
+}
+
+type ElapsedTime func(functionName string) func()
+
+type ChangesInfo struct {
+	Hash           string
+	AuthorName     string
+	AuthorEmail    string
+	Message        string
+	CurrentVersion string
+	NewVersion     string
+	ChangeType     string
+}
+
+type UpgradeFiles struct {
+	Files []UpgradeFile
+}
+
+type UpgradeFile struct {
+	Path            string
+	DestinationPath string
+	VariableName    string
+}
+
+type FileVersion struct {
+	log                Logger
+	elapsedTime        ElapsedTime
+	versionConrolHost  string
+	repositoryRootPath string
+	groupName          string
+	projectName        string
+}
+
+func (f *FileVersion) openFile(filePath string) (*os.File, error) {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return nil, err
+	}
+	return file, nil
+}
+
+func (f *FileVersion) findMessageTag(commitMessage string) bool {
+	return strings.Contains(strings.ToLower(commitMessage), messageTag)
+}
+
+func (f *FileVersion) getMessage(messageRow string) (string, error) {
+	startPosition := strings.Index(messageRow, messageTag) + len(messageTag)
+
+	if startPosition-1 == len(messageRow)-1 {
+		return "", errors.New("message not found")
+	}
+
+	message := strings.TrimSpace(messageRow[startPosition:])
+	if strings.ReplaceAll(message, " ", "") == "" {
+		return "", errors.New("message not found")
+	}
+
+	return message, nil
+}
+
+// pretiffy aims to keep a short message based on the commit message, removing extra information such as commit type.
+// Args:
+// 		commitMessage (string): Full commit message.
+// Returns:
+// 		string: Returns a commit message with limmited number of characters.
+// 		err: Error whenever unexpected issues happen.
+func (f *FileVersion) prettifyCommitMessage(commitMessage string) (string, error) {
+
+	var message string
+	splitedMessage := strings.Split(commitMessage, "\n")
+
+	for _, row := range splitedMessage {
+		row := strings.ToLower(row)
+		if f.findMessageTag(row) {
+
+			currentMessage, err := f.getMessage(row)
+			if err != nil {
+				return "", err
+			}
+			message = currentMessage
+		}
+	}
+
+	if message == "" {
+		return "", errors.New("commit message has no tag 'message:'")
+	}
+	// Limmit message to 150 characters to avoid long messages.
```

Comment:
instead use one comment here.. what do you think create one function like:

    checkMsgSizeLimit